src/one_dragon/base/operation/one_dragon_context.py

```
@@ -80,7 +79,7 @@ def __init__(self):
 
         self.keyboard_controller = keyboard.Controller()
         self.btn_listener = PcButtonListener(on_button_tap=self._on_key_press, listen_keyboard=True, listen_mouse=True)
-        self.btn_listener.start()
+        # 推迟启动：不要在 __init__ 早期启动钩子，避免初始化/加载重型 C 扩展库时导致鼠标严重卡顿
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)